packages/ts/src/components/treemap/style.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/css'
+
+// Utils
+import { getCssVarNames, injectGlobalCssVariables } from 'utils/style'
+
+const cssVarDefaults = {
+  '--vis-treemap-tile-stroke-color': '#fff',
+  '--vis-treemap-tile-stroke-width': '2px',
+  '--vis-treemap-tile-hover-stroke-color': '#fff',
+  '--vis-treemap-tile-fill-color': '#B9BEC3',
+  '--vis-treemap-tile-background-color': '#fff',
+  '--vis-treemap-tile-cursor': 'default',
+  '--vis-treemap-label-text-color': '#000',
+  '--vis-treemap-label-font-size': '12px',
+
+  /* Label opacity */
+  '--vis-treemap-label-opacity': 0.8,
+
+  /* Dark Theme */
+  '--vis-dark-treemap-tile-stroke-color': '#2c2c2c',
+  '--vis-dark-treemap-tile-fill-color': '#5b5f6d',
+  '--vis-dark-treemap-label-text-color': '#5b5f6d',
+}
+
+export const root = css`
+  label: treemap-component;
+`
+
+export const variables = getCssVarNames(cssVarDefaults)
+injectGlobalCssVariables(cssVarDefaults, root)
+
+export const tiles = css`
+  label: g-tiles;
+`
+
+export const tileGroup = css`
+  label: tile-group;
+`
+
+export const tile = css`
+  label: tile;
+  stroke: var(--vis-treemap-tile-hover-stroke-color);
+  stroke-opacity: 0;
+
+  &:hover {
+    stroke-opacity: 1;
```

Comment:
Not everyone needs interactivity, so I would add a CSS variable for this and made it 0 by default to avoid the unnecessary highlight:

https://github.com/user-attachments/assets/01d44fd7-4a61-4134-8a4d-a25b9de1ceca